com.github.taburineagle.NeewerLite-Python.yaml

```
@@ -0,0 +1,123 @@
+id: com.github.taburineagle.NeewerLite-Python
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
```

```suggestion
runtime-version: 5.15-22.08
```

This is now the newest runtime.